task-definition.plugins.json

```diff
@@ -99,6 +99,10 @@
                 {
                     "name": "KAFKA_PARTITIONS_CONSUMED_CONCURRENTLY",
                     "value": "8"
+                },
+                {
+                    "name": "HISTORICAL_EXPORTS_ENABLED",
+                    "value": "False"
```

Comment:
shouldn't this be True atm?

Reply:
Nobody is complaining and we haven't had a chance to go deeply into it so while I'm reasonably sure it works, I'd like to keep it off for now

Reply:
ok, I didn't know it was off atm sounds good